c3pzero_ignition/launch/spawn_robot.launch.py

```diff
@@ -0,0 +1,228 @@
+            "robot",
+            "-topic",
+            "robot_description",
+            "-z",
+            "1.5",
+        ],
+        output="screen",
+    )
+
+    # IMU Sensor Static Tf
+    imu_static_tf_node = Node(
+        package="tf2_ros",
+        executable="static_transform_publisher",
+        name="imu_static_transform_publisher",
+        output="log",
+        arguments=[
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "base_link",
+            "robot/base_link/imu",
+        ],
+    )
+
+    lidar_static_tf_node = Node(
+        package="tf2_ros",
+        executable="static_transform_publisher",
+        name="imu_static_transform_publisher",
+        output="log",
+        arguments=[
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "0.0",
+            "lidar_link",
+            "robot/base_link/lidar",
+        ],
+    )
+
+    ign_ros_bridge_node = Node(
+        package="ros_ign_bridge",
+        executable="parameter_bridge",
+        arguments=[
+            # cmd_vel bridge (ROS2 -> IGN)
+            '/cmd_vel@geometry_msgs/msg/Twist]ignition.msgs.Twist',
+            # Clock (IGN -> ROS2)
+            "/clock@rosgraph_msgs/msg/Clock[ignition.msgs.Clock",
+            # Lidar (IGN -> ROS2)
+            '/scan@sensor_msgs/msg/LaserScan[ignition.msgs.LaserScan',
+            '/scan/points@sensor_msgs/msg/PointCloud2[ignition.msgs.PointCloudPacked',
+            # Joint states (IGN -> ROS2)
+            '/world/default/model/robot/joint_state@sensor_msgs/msg/JointState[ignition.msgs.Model',
+            # IMU (IGN -> ROS2)
+            "/imu@sensor_msgs/msg/Imu[ignition.msgs.IMU",
+            # Ground Truth Odometry (IGN -> ROS2)
+            "/odom@nav_msgs/msg/Odometry[ignition.msgs.Odometry",
+            # Wheel Odometry (IGN -> ROS2)
+            "/wheel/odom@nav_msgs/msg/Odometry[ignition.msgs.Odometry",
+        ],
+        remappings=[
+            ("/world/default/model/robot/joint_state", "joint_states"),
+            ("/imu", "imu/data"),
+        ],
+        output="screen",
+    )
+
+    # Broadcast tf from odom
+    tf_broadcaster_node = Node(
+        package="c3pzero_ignition",
+        executable="odom2tf_broadcaster",
+        name="odom2tf_broadcaster",
+        output="screen",
+    )
+
+    rviz_node = Node(
+        package="rviz2",
+        executable="rviz2",
+        name="rviz2",
+        output="log",
+        arguments=["-d", rviz_config_file],
+        condition=IfCondition(rviz),
+    )
+
+    ros2_controllers_path = os.path.join(
+        pkg_robot_description,
+        "config",
+        "c3pzero_controllers.yaml",
+    )
+    ros2_control_node = Node(
+        package="controller_manager",
+        executable="ros2_control_node",
+        parameters=[robot_description, ros2_controllers_path],
+        output="both",
+    )
```

Comment:
When we launch Studio it also brings up the controller_manager node and spawns controllers based on settings provided in the base config package's `config/base_config.yaml` file, so we should keep controller bringup separate from initialization of the sim environment.